fabrictestbed_extensions/fablib/slice.py

```diff
@@ -2203,6 +2220,21 @@ def submit(
 
         :return: slice_id
         """
+        # Use kwargs to update only the arguments provided
```

Comment:
None of this needs to be here - this is what passing in **kwargs from the caller already does.  `submit()` itself should _not_ have a **kwargs argument (calling `submit(arg1, arg2, **kwargs)` from the caller is expanded into the keyword arguments before this code sees them, so they're already resolved).